### PR TITLE
Complete and Enhance the Home Page for Netflix Clone( DONE)

### LINKED ISSUE #791: [Feature Request]: Complete and Enhance the Home Page for Netflix Clone
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

Description:
The current home page of the Netflix clone is incomplete and lacks the necessary UI elements and styling to provide a fully functional and visually appealing user experience. This issue aims to complete the home page by adding the following features:

Tasks:

Hero Section:
Movie/TV Show Categories:
Navigation Bar:
Footer:
Responsive Design:

### Use Case

Netflix Clone

### Benefits

Improved User Experience
Responsive Design
Enhanced Functionality

### Add ScreenShots

image
image
i can make it look like this pls assign this to me 

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [ ] I'm a GSSOC'24 contributor
- [X] I'm a SSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by shreya-2410**
# Title and Issue number 

Title :Complete and Enhance the Home Page for Netflix Clone #791 

Issue No. :#791

Code Stack : html and css

Close #791 

# Video (mandatory)

Uploading bandicam 2024-06-16 22-00-30-935.mp4…

# Checklist:

- [x] I have mentioned the issue number in my Pull Request.
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have created a helpful and easy to understand `README.md`
- [x] I have gone through the  `contributing.md` file before contributing

**Additional context (Mandatory )**
it is my time contributing hence i could not put readme file but be assured code is easy to understand and have tried my best to clone original netflix ui.

***Are you contributing under any Open-source programme?***

- [ ] I'm a GSSOC'24 contributor
- [x] I'm a SSOC'24 contributor

### CODE REVIEW

**Comment by vHarshita15** [comment]
Want to work on this issue . Plese assign

File: Projects/Netflix_clone_Project/index.js
Comment (apu52) [inline Projects/Netflix_clone_Project/index.js:1]: @shreya-2410 Why this full JS file has been removed & I also can see there is a massive UI change for the Project . Can u please share a video that how many additional features iu have added in this project?? 
Please attach a video with this pR

**Review by apu52 — CHANGES_REQUESTED** [review]

**Comment by shreya-2410** [comment]
https://github.com/apu52/METAVERSE/assets/137192044/e4dff513-4bad-4791-a56b-2b3063771c1f

here is the video of the clone before vs after ..actually i have tried my best enhancing the clones ui ...i have not made it functional by using js ..but changed its appearance completely and hence to avoid any glitches i have removed js file ...

**Review by apu52 — APPROVED** [review]
@shreya-2410 okk

**Comment by shreya-2410** [comment]
why are my points not reflecting on oslead website

**Comment by apu52** [comment]
> why are my points not reflecting on oslead website

I have already added your points & it has been successfully added Can u please check now??

**Comment by apu52** [comment]
image

here at rank 7

**Pull request merged**